### ISSUE TITLE
iOS 14.0 Support

### ISSUE THREAD

**Issue by wwasrgb** [zzanehip/The-OldOS-Project#37]
![600EC33B-0961-4522-B610-3A40379DD730](https://user-images.githubusercontent.com/85804207/121785651-fcab2f00-cb88-11eb-891f-6a7c7b08b037.jpeg)

Does this work on iOS 14.0

**Comment by TheBalkanDev** [comment]
Yes SwiftUI is since YIOS 13 and was improved in iOS 14 with iOS is mean 14.0.0-14.x.x
